lib/core/include/irods_query.hpp

```diff
@@ -106,7 +106,7 @@ namespace irods {
                 return query_string_;
             }
 
-            bool page_in_flight(const int row_idx_) {
+            bool page_in_flight(const int& row_idx_) {
```

Comment:
I don't think the `&` is really needed.  Creating a copy seems better, but the `const` pretty much achieves the same result.

Reply:
Got it. Thanks